sky/adaptors/azure.py

```diff
@@ -15,18 +17,31 @@
 _session_creation_lock = threading.RLock()
 
 
-@common.load_lazy_modules(modules=_LAZY_MODULES)
+def _run_output(cmd):
+    proc = subprocess.run(cmd,
+                          shell=True,
+                          check=True,
+                          stderr=subprocess.PIPE,
+                          stdout=subprocess.PIPE)
+    return proc.stdout.decode('ascii')
```

Comment:
We actually have an util function for this:
https://github.com/skypilot-org/skypilot/blob/f3be7732a1d9890e24dd9d87f9bdc19faeeeca8b/sky/utils/subprocess_utils.py#L21-L35